src/itkCudaDataManager.cxx

```diff
@@ -24,7 +24,6 @@ namespace itk
 // constructor
 CudaDataManager::CudaDataManager()
 {
-  m_ContextManager = CudaContextManager::GetInstance();
 
```

Comment:
Can you remove this long comment too? I think it was already obsolete. Sorry, I should have caught this before...

Reply:
Done